e2e/21_create-pages_standalone.spec.ts

```diff
@@ -32,6 +32,12 @@ async function testRouterExample(page: Page, port: number) {
   await page.goto(`http://localhost:${port}/foo`);
   await expect(page.getByRole('heading', { name: 'Foo' })).toBeVisible();
 
+  // /nested/foo is defined as a staticPath of /nested/[id] which matches this layout
+  await page.goto(`http://localhost:${port}/nested/foo`);
+  await expect(
+    page.getByRole('heading', { name: 'Deeply Nested Layout' }),
+  ).toBeVisible();
+
```

Comment:
That was my bad too, but it's actually not a good idea to rely on examples. This is kind of historical reason, but we now have some standalone spec with fixtures.

Let's revisit it with a different PR later.